holo-routing/src/lib.rs

```diff
@@ -89,24 +89,28 @@ impl Master {
 
         loop {
             tokio::select! {
-                Some(request) = nb_rx.recv() => {
-                    process_northbound_msg(
-                        self,
-                        &mut resources,
-                        request,
-                    )
-                    .await;
-                }
-                Some(msg) = ibus_rx.recv() => {
-                    ibus::process_msg(self, msg);
-                }
-                Some(_) = self.rib.update_queue_rx.recv() => {
-                    self.rib
-                        .process_rib_update_queue(
-                            &self.netlink_handle,
-                        )
-                        .await;
-                }
+               Some(request) = nb_rx.recv() => {
```

Comment:
nit: unnecessary whitespace changes (unfortunately *rustfmt* ignores everything within the `select!` macro so we need to take care of this ourselves).

Reply:
Fixed in 26eb5745426fe3a042a1b0a2cabd33d18633449c.